www/js/control/ProfileSettings.jsx

```diff
@@ -156,7 +156,12 @@ const ProfileSettings = () => {
 
     //ensure ui table updated when editor closes
     useEffect(() => {
-        refreshCollectSettings();
+        if(editCollection == false) {
+            setTimeout(function() {
+                console.log("closed editor, time to refresh collect");
+                refreshCollectSettings();
+              }, 1000);
+        }
     }, [editCollection])
```

Comment:
I'm a bit confused. Where is `editCollection` changed? I only see `setEditVis` called here.

```
        try{
            let set = await setConfig(localConfig);
            setEditVis(false);
        } catch(err) {
            Logger.displayError("Error while setting collection config", err);
        }
```

Reply:
The variable name is different in the different files, but I realize the name in `ProfileSettings` was confusing and did not match the rest of the file. I had been consistently using "Vis" in the state names for visibility, and should do that here. I've updated it to `editCollectionVis`.